xray/client/package.json

```diff
@@ -5,16 +5,25 @@
   "main": "index.js",
   "scripts": {
     "build": "rollup -c",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
Did prettier put that line here? Remove?

Reply:
No, it was there before, I just changed the ordering to build/test/format, whereas it was build/format/test before. Probably makes sense to just remove it though.

Reply:
Done.